src/components/Search/Search.tsx

```diff
@@ -35,6 +45,12 @@ const Search = (): JSX.Element => {
     setSearchValue(event.target.value);
   };
 
+  const handleKeyEvent = (event: React.KeyboardEvent<HTMLElement>): void => {
+    if (event.key === 'Enter') {
```

Comment:
for handling Enter you can wrap your inputs in form => its onSubmit event by default watch for Enter keypress

Reply:
added form wrapper